aws/aws.go

```diff
@@ -135,3 +142,55 @@ func NewSessionFromURLParams(q url.Values) (*session.Session, url.Values, error)
 	}
 	return sess, rest, nil
 }
+
+// UseV2 returns true iff the URL parameters indicate that the provider
+// should use the AWS SDK v2.
+//
+// "awssdk=v1" will force V1.
+// "asssdk=v2" will force V2.
+// No "awssdk" parameter (or any other value) will return the default, currently V1.
+func UseV2(q url.Values) bool {
+	for param, values := range q {
+		if param == "awssdk" {
+			if values[0] == "v2" || values[0] == "V2" {
+				return true
+			}
+			return false
+		}
+	}
+	return false
```

Comment:
Rather than looping over all the values, this may be simplified to a single parameter lookup, like so:
```suggestion
if values, ok := q["awssdk"]; ok {
	if values[0] == "v2" || values[0] == "V2" {
		return true
	}
}
return false
```

Reply:
Thanks, done!